docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/06_Inserting_Images.md

```diff
@@ -20,14 +20,29 @@ Image files for a website must be in either JPG, GIF or PNG format. If your imag
 
 ## Setting the alignment of an image
 
-1. Select the image and click the button **Insert from files** shown as a file icon in the HTML editor.
+1. Select the image and click the button ***Insert from files*** shown as a file icon in the HTML editor.
 2. Navigate to the ***Placement*** tab.
-3. Select the alignment for your image from the **Alignment** dropdown field.
+3. Select the alignment for your image from the ***Alignment*** dropdown field.
 * Left
 * Center
 * Left or right wrap
 4. Click the ***Insert file*** button.
 
 ## Repositioning an Image
 
-To change the position of an image that you have inserted, simply click and drag it, moving the text cursor to the position you want to move the image to. Remember, the text cursor position only defines the position of the top of the image, so you're best dragging it to the very start of a paragraph, or onto it's own line.
+To change the position of an image that you have inserted, simply click and drag it, moving the text cursor to the position you want to move the image to. Remember, the text cursor position only defines the position of the top of the image, so you're best dragging it to the very start of a paragraph, or onto its own line.
+
+## Controlling the loading of images {#lazy-loading}
+
+Modern web browsers can delay the loading of images until the image is about to be displayed to the user. This feature is called "lazy loading", as opposed to "eager loading" where the browser downloads images on page load even if the user can not see them. 
+
+Lazy loading reduces the time it takes display the page to users. By default, Silverstripe CMS will lazy load any image you insert into an HTML editor.
+
+If an image is placed near the top of the page and you expect it to be immediately visible to users, you may wish to eager load it.
+
+To force an image in an HTML editor to eager load:
+
+1. Select the image and click the button ***Insert from files*** shown as a file icon in the HTML editor.
+2. Navigate to the ***Placement*** tab.
```

Comment:
The placement tab is the default tab, so we can probably omit this point

Reply:
It's only the default tab when coming from the editor, otherwise the default tab is 'Details'. How about just making a lead into the following step... `Within the placement tab, select the alignment for your image from the ***Alignment*** dropdown field.`

Reply:
There isn't a "placement" tab anymore. When we added the ability to edit files in-modal, we move all the other tabs to the detail view. Looks like we forgot to update the doc when we did that work.

I'll remove reference to the "placement" tab and review the rest of the article to make sure we accounh for the in-modal editing work.